### DISCUSSION TITLE
Video contribution healthcheck

### DISCUSSION THREAD

**Discussion by ciroiriarte** [OvenMediaLabs/OvenMediaEngine#1430] [Help]
Hello!, I just found out about OME and would like to build a PoC.

Trying to wrap the architecture around my head, I wonder if OME can run a basic healthcheck on the contributed streams at ingress time to detect issues like artifacts. If that's not the case, I should consider some kind of external middleman for that.

The idea is to reject bad quality input and avoid doing any work on that stream untill it comes back with acceptable quality for distribution.

**Comment by bchah** [comment]
Have you tried the Alert feature? It can send a webhook when certain thresholds are passed:

https://airensoft.gitbook.io/ovenmediaengine/alert
